config.json

```diff
@@ -2250,6 +2250,22 @@
         ],
         "difficulty": 5
       },
+      {
+        "slug": "killer-sudoku-helper",
+        "name": "Killer Sudoku Helper",
+        "uuid": "ac7f32a8-920b-4132-b904-76131b344f12",
+        "prerequisites": [
+          "integers",
+          "lists",
+          "ranges",
+          "enum",
+          "maps",
+          "pattern-matching",
+          "recursion"
+        ],
+        "practices": [],
+        "difficulty": 6
```

Comment:
OK, I agree.
Originally I placed it by `knapsack` because the problem is very very similar, however for this one, brute force is the best option since the size of the digits to chose is so constrained.